tests/nested/core-revert/task.yaml

```diff
@@ -45,28 +43,36 @@ execute: |
     execute_remote "snap info core" | MATCH "tracking: +${CORE_CHANNEL}"
     execute_remote "sudo snap refresh --${CORE_REFRESH_CHANNEL} core" || true
 
-    wait_for_ssh
+    if ! wait_for_ssh; then
+        echo "ssh not stablished, exiting..."
+        exit 1
+    fi
 
-    while ! execute_remote "snap changes" | MATCH "Done.*Refresh \"core\" snap from \"${CORE_REFRESH_CHANNEL}\" channel"; do sleep 1 ; done
+    while ! execute_remote "snap changes" | MATCH "Done.*Refresh \"core\" snap from \"${CORE_REFRESH_CHANNEL}\" channel"; do
+        sleep 1
+    done
     execute_remote "snap info core" | MATCH "tracking: +${CORE_REFRESH_CHANNEL}"
     # make sure network-manager is on charge of eth0
     execute_remote "nmcli c" | MATCH eth0
     execute_remote "nmcli d" | MATCH "eth0 +ethernet +connected"
-    execute_remote "sudo bluez.bluetoothctl -a"
+
     # sanity check, no refresh should be done here but the command shouldn't fail
     execute_remote "sudo snap refresh"
 
     execute_remote "snap aliases" | MATCH "nmcli"
 
     execute_remote "sudo snap revert core" || true
 
-    wait_for_ssh
+    if ! wait_for_ssh; then
+        echo "ssh not stablished, exiting..."
+        exit 1
+    fi
 
     while ! execute_remote "snap changes" | MATCH "Done.*Revert \"core\" snap"; do sleep 1 ; done
     execute_remote "snap aliases" | MATCH "nmcli"
     execute_remote "snap info core" | MATCH "tracking: +${CORE_REFRESH_CHANNEL}"
     execute_remote "ifconfig" | MATCH eth0
-    execute_remote "sudo bluez.bluetoothctl -a"
+    #execute_remote "sudo bluez.bluetoothctl -a"
```

Comment:
Given that similar bluez installs were removed previously, why is this kept (and commented out) instead of removed?